### PR TITLE
New/global footer

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

File: src/core/_core.variables.scss [hunk 1]
```diff
@@ -114,7 +114,7 @@ $fhi-yellow-1: #fffdce;
 $fhi-orange-1: #feeed9;
 $fhi-orange-2: #f7b665;
 
-$fhi-red-1: #fd7065;
+$fhi-red-1: #f86559;
```
Comment (proand) [inline src/core/_core.variables.scss:117]: I think we should defer color changes to v4

File: src/icons/_icon-common-style.scss [hunk 2]
```diff
@@ -11,3 +11,7 @@
 .icon {
     @include icon-classes-generator(); // (mixin)
 }
+
+.icon-white {
+    filter: brightness(1000%);
```
Comment (proand) [inline src/icons/_icon-common-style.scss:16]: Hmm... this is probably intended, but the code looks like at bug; is there another way to do this?

**Comment by proand** [comment]
Talked to Cathrine; we don't animate the footer ;)

File: src/blocks/app-and-main-menu/_fhi-footer.block.scss [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    &::after {
+        bottom: 0;
+        height: 100%;
+        left: 0;
+        right: 0;
+        top: ($fhi-space-6 * 2);
+        transition: all .2s;
+    }
+
+    &__container {
```
Comment (proand) [inline src/blocks/app-and-main-menu/_fhi-footer.block.scss:46]: I rather you do `.fhi-footer__container` inside the modifier, and skip `fhi-footer--arch__container` in the markup

**Review by proand — CHANGES_REQUESTED** [review]

File: src/icons/_icon-common-style.scss (hunk 2 above)
Comment (yleaxeman) [reply src/icons/_icon-common-style.scss:16]: There's always other ways, but only more cumbersome ones...

Comment (proand) [reply src/icons/_icon-common-style.scss:16]: Ok, maybe drop in [sic]-comment then ;)

**Review by proand — APPROVED** [review]
👍

**Pull request merged**